cetmix_tower_aws/readme/CONFIGURE.md

```
@@ -0,0 +1,40 @@
+## AWS Configuration
+
+### Setting up AWS Access
+
+1. **Create AWS Access Keys**
+
+   To use the AWS integration with Cetmix Tower, you need to create AWS access keys:
+   
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)